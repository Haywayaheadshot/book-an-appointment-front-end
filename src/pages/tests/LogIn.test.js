// import React from 'react';
// import { render } from '@testing-library/react';
// import { MemoryRouter } from 'react-router-dom';
// import LogIn from '../LogIn';

// describe('LogIn', () => {
//   it('renders correctly', () => {
//     const { container } = render(
//       <MemoryRouter>
//         <LogIn />
//       </MemoryRouter>
//     );
//     expect(container).toMatchSnapshot();
//   });
// });
