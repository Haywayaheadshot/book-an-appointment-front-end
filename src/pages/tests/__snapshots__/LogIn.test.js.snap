// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogIn renders correctly 1`] = `ShallowWrapper {}`;
